src/styles/_catalog.scss

```diff
@@ -1,22 +1,31 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: 48px;
+  justify-content: center;
+  grid-template-columns: repeat(auto-fill,200px);
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (max-width: 488px) {
+    grid-template-columns: 200px;
+  }
```

Comment:
It's a redundant query, because you have already set grid-template-colums above 
```suggestion

```

Reply:
![image](https://user-images.githubusercontent.com/116836849/206243169-b7dd9329-6887-481c-b05d-34625d5138ce.png)
  - There should be one column below `488px`

without this query we will have this:

![image](https://user-images.githubusercontent.com/116836849/206243675-a017979a-a8b2-48d8-ad3d-370621e00de0.png)

Reply:
Ok, so in this case you do not need the rule grid-template-columns: repeat(auto-fill,200px), because you overwrite it by this query